templates/errors/401.html

```diff
@@ -0,0 +1,6 @@
+{% extends 'layouts/_base.html' %}
+
+{% block main %}
+<h1>{{ _("Sorry, there is a problem") }}</h1>
+<p>{{ _("To access this page you need to <a href='#'>enter your unique access code.</a>") }}</p>
+{% endblock %}
```

Comment:
I don't think this template is used, as 401 errors seem to use the `session-expired` template?